docs/advanced-node-scheduling.asciidoc

```diff
@@ -0,0 +1,342 @@
+[id="{p}-advanced-node-scheduling"]
+
+== Advanced Elasticsearch node scheduling
```

Comment:
Would "Advanced scheduling on Elasticsearch node" sound any better?

Reply:
It's about scheduling multiple Elasticsearch nodes. Maybe I should use the plural of "nodes"?

Reply:
I think this title is absolutely fine.